tests/config/operator_install/olsconfig.crd.azure_openai_introspection.yaml

```diff
@@ -34,3 +34,10 @@ spec:
       - name: bar_filter
         pattern: '\b(?:bar)\b'
         replaceWith: "openshift"
+mcp_servers:
```

Comment:
isn't this meant to be under spec? is the tabbing correct?

Reply:
yes...